src/cookbooks/clients_last_seen_bits.md

```diff
@@ -0,0 +1,136 @@
+# Working with Bit Patterns in Clients Last Seen
+
+An powerful feature of the `clients_last_seen` methodology is that we can capture
+per-day usage information for the entire 28-day period that each row in the table
+describes. We encode usage information in "bit patterns" where the physical
+type of the field is a BigQuery 64-bit integer, but logically the integer
+represents an array of bits, with each 1 indicating a day where the given clients
```

Comment:
How is the array ordered?

Reply:
I'm hoping this is addressed by the new "Understanding bit patterns" section, but let me know if it still feels like that needs to be addressed here.

Reply:
Yes, this is clear below.